server/notification-providers/slack.js

```diff
@@ -48,7 +48,7 @@ class Slack extends NotificationProvider {
 
         }
 
-        if (monitorJSON.url) {
+        if (this.extractAdress(monitorJSON)) {
```

Comment:
the extracted `adress` should be used below instead of `monitorJSON.url`.

(I can't suggest this change because of github restrictions)

```suggestion
        const adress = this.extractAdress(monitorJSON);
        if (adress) {
```

Reply:
My bad :facepalm: